README.md

```diff
@@ -12,10 +12,10 @@ NOTICE,  SkyWalking Client JS 0.8.0 and later versions require SkyWalking v9.
 # Usage
 
 ## Install  
-The `skywalking-client-js` runtime library is available at [npm](https://www.npmjs.com/package/skywalking-client-js).
+The `skywalking-client-js` runtime library is available at [yarn]([https://classic.yarnpkg.com/en/package/webpack/skywalking-client-js)].
```

Comment:
Which yarn version do we need?

Reply:
in my local, i use version is 1.22.11 , it can effective for webpack

Reply:
This is a doc for end users and developers, it should be very clear which versions(such as 1.22.x+ or something).

Reply:
Thanks, Let me add it.